components/src/core/score-card/score-card.tsx

```diff
@@ -183,36 +183,36 @@ const HeaderText: React.FC<HeaderTextProps> = (props) => {
     const textColor = color || 'white';
     return (
         <View style={{ flex: 1 }}>
-            <Typography.H7
+            <Typography.H6
```

Comment:
the fontsize here in the scorecard needs to be updated to 18...H6 default is too large

Reply:
Is it ok to add a "mediumLarge" variant with a size of 18 to the sizes/index.ts file?